src/pages/Revenue/Revenue.component.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { revenueActions } from './Revenue.action';
+import Menu from '../../components/Menu/index';
+import TableWrapper from '../../globalstyles.js';
+import RevenueTableWrapper from './styled';
+
+class RevenueComp extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            data: {}
+        }
+    }
+
+    componentDidMount() {
+        var revenueData = revenueActions.getAllCompany();
+        this.setState({ data: revenueData });
+    }
+
+    showCompanyRevenue(id) {
+        this.props.history.push(`/revenue/${id}`);
+    }
+
+    render() {
+        let { data } = this.state;
+        return (
+            <div>
+                <Menu selected={"revenue"} />
+                <RevenueTableWrapper>
+                    <TableWrapper>
+                        <table>
+                            <thead>
+                                <tr>
+                                    <td>Company Name </td>
```

Comment:
we cannot hardcode column name. We should define a table component and pass this as props